ihatemoney/forms.py

```diff
@@ -139,6 +139,7 @@ class BillForm(FlaskForm):
     what = StringField(_("What?"), validators=[DataRequired()])
     payer = SelectField(_("Payer"), validators=[DataRequired()], coerce=int)
     amount = CalculatorStringField(_("Amount paid"), validators=[DataRequired()])
+    document = URLField(_("Document"), validators=[Optional()], description="A link to your bill.")
```

Comment:
The description here isn't clear enough. What do you think about "A link to an external document, related to this bill"

Reply:
Also, if this is a link, I feel that the name here isn't right, because it isn't explicit about what is stored (it's not a document, it's an URL, or a link). What about ``external_link`` or ``document_url` rather than "document"?

Reply:
I don't mind just let me know what you think it's the best for your project :smile:.
I'll change the description and the field name for external_link.
Just a remark, I don't know how to show that it's an optional field, it looks like other that are required and how to display the html5 field check error. If you have an hint tell me :loudspeaker:

Reply:
let's go for ``external_link`` then. You can put ``(Optional)`` in plain text, I think?